src/plasma/plasma_store.c

```diff
@@ -359,6 +363,59 @@ void remove_objects(plasma_store_state *plasma_state,
   }
 }
 
+/* Convert object_id to a unique file path on-disk */
+char *object_id_to_persist_path(object_id object_id) {
```

Comment:
please use the function sha1_to_hex http://github.com/ray-project/ray/blob/master/src/common/common.h#L118